src/App.tsx

```diff
@@ -17,18 +37,30 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter onFilter={setSort} />
```

Comment:
Ensure that the `TodoFilter` component correctly handles the `onFilter` prop to update the filter state. The `statusOptions` in `TodoFilter` has a typo: 'lable' should be 'label'. This could affect the display of options in the dropdown.